net/https-dns-proxy/files/https-dns-proxy.init

```diff
@@ -1,9 +1,17 @@
 #!/bin/sh /etc/rc.common
-# Copyright 2019 Stan Grishin (stangri@melmac.net)
+# Copyright 2019-2020 Stan Grishin (stangri@melmac.net)
 # shellcheck disable=SC2039
 
-export START=80
-export USE_PROCD=1
+readonly PKG_VERSION='2020-08-21-1'
```

Comment:
Please don't do this. If you forget to bump this here users go bananas. 

Please use something like `readonly PKG_VERSION='%PKG_VERSION%'` and add to `Package/https-dns-proxy/install` the following step `$(SED) "s;%PKG_VERSION%;$(PKG_VERSION);" $(1)/etc/init.d/https-dns-proxy`

Reply:
Done. That was what I was doing in `build/prepare` before.